game/itembuilds/default_dragon_knight.txt

```diff
@@ -9,6 +9,13 @@
 		"#DOTA_Item_Build_Starting_Items"
 		{
             "item" "item_tango"
+            "item"  "item_demon_edge"
+            "item"  "item_lesser_crit"
+            "item"  "item_recipe_greater_crit"
 		}
+
+        "#DOTA_Item_Build_Late_Items"
```

Comment:
Need to move the items to this row after making it possible to access it via UI highlight paths.